commands/service_docs.go

```diff
@@ -26,15 +26,17 @@ func newServiceDocsCmd(e ServiceExecutor) *serviceDocsCmd {
 		Use:   "gen-doc",
 		Short: "Generate the documentation for the service in a README.md file",
 		Example: `mesg-core service gen-doc
-mesg-core service gen-doc ./PATH_TO_SERVICE`,
+mesg-core service gen-doc ./PATH_TO_SERVICE
+mesg-core service gen-doc --force`,
 		PreRunE: c.preRunE,
 		RunE:    c.runE,
 	})
+	c.cmd.Flags().BoolVarP(&c.force, "force", "f", c.force, "No confirmation. Will replace existing README.md file")
```

Comment:
I don't see the use of this flag in the command

Reply:
https://github.com/mesg-foundation/core/pull/823/files#diff-aaa5128dba1f3dafaedf0d439b020430R41

Reply:
The logic was already implemented but the flag was not exposed

Reply:
I think having force option is good for non interactive terminals. But we can change it to `--yes` because we use it in general, in other command too. It exists for answering all terminal prompts as yes by default in a command action. But this improvement can be done in an another PR.